emscripten.py

```diff
@@ -691,7 +694,8 @@ def all_asm_consts(metadata):
   all_sigs = []
   for k, v in metadata['asmConsts'].iteritems():
     const = v[0].encode('utf-8')
-    sigs = v[1]
+    call_types = v[1][0::2]
```

Comment:
That is uh, non-obvious
Instead of interleaving signature and calltype, can we make them separate sub-arrays? So this can be just:
```python
call_types = v[1]
sigs = v[2]
```

Reply:
Also apparently I missed this obvious step back in https://github.com/kripken/emscripten/pull/5137, and `create_asm_consts_wasm` isn't calling `all_asm_consts`, which would make our lives easier.

Reply:
> Instead of interleaving signature and calltype, can we make them separate sub-arrays?

Ok, that is also backwards compatible, so probably better.